docs/user/integration/integrate-jaeger.md

```diff
@@ -0,0 +1,181 @@
+# Install custom Jaeger in Kyma
+
+## Overview
```

Comment:
Currently we have this file structure:
- integrate-loki.MD
- integrate-loki
  - asset-files

What about just moving the MD file into the folder and call it README?

Reply:
Because "README" as a file name is completely void of meaning. I prefer titles that tell me what a document is about, without having to look at the folder name.

Reply:
Done